### ISSUE TITLE
Is there a pretrained model available?

### ISSUE THREAD

**Issue by Dyex719** [kirel/detexify-hs-backend#1]
I am trying to improve this to suit medical data, it would be really helpful if a pre-trained model is available.

**Comment by Chaostheorie** [comment]
I don't think so though, even if this request is probably stale by now, [see this repository for instructions on how to obtain and use the training data](https://github.com/kirel/detexify-data).
